frontend/src/app/Settings/Users/ManageUsers.scss

```diff
@@ -229,7 +229,7 @@
     }
 
     .disabled-dark * {
-      color: #adadad !important;
+      color: #6e6e6e !important;
```

Comment:
Would one of the [gray colors ](https://designsystem.digital.gov/utilities/color/)within the uswds design system work for this fix so we can follow the the design system guidance 🤔 ?. Unless the `6e6e6e` color used here was specifically asked by the ux team

Reply:
This was not specifically called out from the ux team. I'll replace it with the closet color in that design system`.text-gray-50 #757575`